src/rechoseWorkflow.ts

```diff
@@ -0,0 +1,126 @@
+import prompts from "prompts";
+import { red } from "kolorist";
+import { validateProjectName } from "./utils/validation.js";
+
+/** workflow object containing all the text for the different prompt options */
+export const workflowOptions = {
+  projectName: {
+    type: "text",
+    name: "projectName",
+    message: "Enter a new project name",
+    validate: (value: string) => validateProjectName(value),
```

Comment:
@0xmaayan Sounds good! The only thing to point out though, is currently the name state is being saved when people go to rename their project, but setting the initial value like this would mean the default would be "my-aptos-app" always and does not keep the previous name. Not too big of a deal, so I will make the change now, but just letting you know of the tradeoff.

Reply:
Oh actually this is a great callout. Lets keep the chosen name then when they want to rechose